RELEASE-NOTES.txt

```diff
@@ -1,10 +1,17 @@
-3.0
+2.11
 -----
- 
+
+2.10
+-----
+* Added improvements for devices with hardware keyboards
+* Added support for screen readers when adding a new note
+* Fixed an issue with font size resizing when swiping
```

Comment:
```suggestion
```

The font resizing issue was fixed, but it's an extreme edge case and didn't cause any functionality problems.  It was just an annoying glitch.  Due to that, I think we can leave it out of the release notes since it doesn't affect the app's usability and a very small fraction of users would see it.